scalyr

```diff
@@ -55,6 +57,33 @@ def getApiToken(args, environmentVariableName, permissionType):
     return apiToken
 
 
+def getProxyAddress(args):
+    proxyAddress = args.proxy
+
+    if proxyAddress is None:
+        # Check if we have a environment variable
+        if os.environ.get('http_proxy'):
```

Comment:
I believe a lot of software uses upper case versions of those environment variables names and IIRC, ``os.environ.get`` is case sensitive (well, on some platform).

So one option would be to handle both scenarios and another one would be to document which formats needs to be used in README - in fact, that's something we should do regardless.